spec/factories/users_setting.rb

```diff
@@ -0,0 +1,15 @@
+FactoryBot.define do
+  factory :users_setting, class: "Users::Setting" do
```

Comment:
I redefined this factory's name due to consistently-failing specs that complained that the factory `users_setting` did not exist. I took this approach since this factory serves just one spec file.

= = =

<img width="1043" alt="Screen Shot 2021-03-30 at 11 46 42 AM" src="https://user-images.githubusercontent.com/32520970/113027058-ac126100-9157-11eb-9326-008bc7c74177.png">